Xamarin.PropertyEditing.Tests/MockPropertyInfo/MockPropertyInfo.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Collections.Generic;
+
+namespace Xamarin.PropertyEditing.Tests.MockPropertyInfo
+{
+	public class MockPropertyInfo<T> : IPropertyInfo
+	{
+		public MockPropertyInfo (string name, string category = "", bool canWrite = true)
+		{
+			Name = name;
+			Category = category;
+			CanWrite = canWrite;
+		}
+
+		public string Name { get; }
+		public Type Type => typeof (T);
```

Comment:
This doesn't work for enums, their representation type is the base type (int, short, etc) for the enum.